config/ldes-delta-pusher/handle-types-util.ts

```diff
@@ -0,0 +1,65 @@
+import { Changeset } from "../types";
+import { query } from "mu";
+import { publish } from "./publisher";
+import { log } from "./logger";
+
+export type InterestingSubject = {
+  uri: string;
+  ldesType: string;
+};
+
+type SubjectsToInterestingSubjects = {
+  (subjects: string[]): Promise<InterestingSubject[]>;
```

Comment:
this feels like a weird type notation. shouldn't this be `(subjects:string[]) => Promise<InterestingSubject[]>`?

Reply:
It can be written as:
```
type SubjectFilter = {
  (subjects: string[]): Promise<InterestingSubject[]>;
};
```
or
```
type SubjectFilter = (subjects: string[]) => Promise<InterestingSubject[]>;
```